content/en/docs/tasks/configmap-secret/managing-secret-using-config-file.md

```diff
@@ -170,7 +170,56 @@ type: Opaque
 
 `YWRtaW5pc3RyYXRvcg==` decodes to `administrator`.
 
-## Clean up
+## Edit a Secret {#edit-secret}
+
+To edit the data in the Secret you created using a configuration
+file, modify the `data` or `stringData` field in your original configuration
+file and apply the file to your cluster. You can edit an existing `Secret` object unless it is [immutable](/docs/concepts/configuration/secret/#secret-immutable).
```

Comment:
```suggestion
To edit the data in the Secret you created using a manifest, modify either
the `data` or `stringData` field in your manifest, and apply the file to your cluster.
You can edit an existing `Secret` object unless it is [immutable](/docs/concepts/configuration/secret/#secret-immutable).
```

Reply:
Separately Tim, do you think we should rename the actual task to "Managing Secrets Using a Manifest"?